module/VuFind/src/VuFind/Db/Service/AccessTokenService.php

```diff
@@ -83,7 +77,19 @@ public function getByIdAndType(
         string $type,
         bool $create = true
     ): ?AccessTokenEntityInterface {
-        return $this->accessTokenTable->getByIdAndType($id, $type, $create);
+
+        $dql = 'SELECT * '
+            . 'FROM ' . $this->getEntityClass(AccessToken::class)
+            . 'WHERE id = :id '
+            . 'AND type = :type';
+        $query = $this->entityManager->createQuery($dql);
+        $query->setParameters(['id' => $id, 'type' => $type]);
+        $result = $query->getResult();
+        $result = $result ?: $this->createEntity()
```

Comment:
I think you may need to account for the `$create` argument here -- return the new object if `$create` is true, but return null if it is false.